src/pages/solve/Knapsack.tsx

```diff
@@ -0,0 +1,39 @@
+import { Heading, Text } from "@chakra-ui/react";
+import { NextPage } from "next";
+import { useState } from "react";
+import { Layout } from "../../components/layout/Layout";
+import { SolverConfiguration } from "../../components/solvers/SolverConfiguration";
+import { TextInputMask } from "../../components/solvers/TextInputMask";
+
+const Knapsack: NextPage = () => {
+  const [knapsackProblem, setKnapsackProblem] = useState("");
+
+  return (
+    <Layout>
+      <Heading as="h1">Knapsack Solver</Heading>
+
+      <Text color="text" align="justify">
+        The Knapsack problem is a combinatorial optimization problem that seeks
```

Comment:
Please clarify in the text that this problem mask is for 0-1 knapsack problems.

Reply:
Mentioned in commit 3456083814448bdfe1986300d9c3dd6195283a68